test-pcapplusplus.sh

```diff
@@ -10,12 +10,13 @@
 # exit when any command fails
 set -e
 
-# This part runs only inside the container - the script runs itelsf inside the container
+# This part runs only inside the container - the script runs itself inside the container
 if [ "$1" == "build_and_run" ]; then
     # Find the required configuration - default ,DPDK or PF_RING
     CONFIG_PARAMS="--default"
     if [ "$2" == "dpdk" ]; then
         CONFIG_PARAMS="--dpdk --dpdk-home /dpdk"
+        FLAGS="-march=native"
```

Comment:
I think it's a good idea to use march=native

Regarding the machine where you build the docker image it will not give the same output :(